CIPs/CIP-19/CIP-19.md

```diff
@@ -74,6 +74,43 @@ The Basic Profile schema defines the format of a document that contains the prop
       "type": "string",
       "pattern": "^ipfs://.+",
       "maxLength": 150
+    },
+    "positiveInteger": {
+      "type": "integer",
+      "minimum": 1
+    },
+    "imageMetadata": {
+      "type": "object",
+      "properties": {
+        "src": {
+          "$ref": "#/definitions/IPFSUrl"
+        },
+        "width": {
+          "$ref": "#/definitions/positiveInteger"
+        },
+        "height": {
+          "$ref": "#/definitions/positiveInteger"
+        },
+        "size": {
+          "$ref": "#/definitions/positiveInteger"
+        }
```

Comment:
Is this the file size? Do we need that?

Reply:
It's not a required field but could be useful for consumer, for example when loading over mobile networks.